### ISSUE TITLE
What do we use to describe "spatial markers' in cells

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#15672]
We have some proteins which function as "cell end markers"
Abstract
i.e. We suggest that tea1 acts as an end marker, directing the growth machinery to the cell poles. tea1 is down-regulated in cells treated with pheromone that grow toward a mating partner and no longer maintain their ends exactly opposed.

Intro
An important problem for a cell is how it organizes itself in space. This organization requires that different com- ponents or points located at a distance within the cell are correctly positioned with respect to each other. The problem is similar in principle to a developmental field establishing positional information during embryogene- sis but acts at the level of a single cell. There are many examples of this phenomenon: epithelial cells have op- posed apical and basal faces with different properties (Eaton and Simons, 1995); Drosophila oocytes contain determinants localized to discrete regions of the cyto- plasm (Lehmann, 1995); and bipolar budding yeast cells bud at the opposite end of the cell to the previous bud (Roemer et al., 1996).

(this is quite an old paper, but I don't think we have ever captured the "molecular role" of this protein ) or similar "spatial markers"

**Comment by mah11** [comment]
On a related note, we would find it useful to have a function or process term to describe how a cell sets the boundary of a region that isn't bounded by a membrane or other physical structure - the cell end again, in our immediate use case. In other words, how does a cell "know" where the border between its end and its medial region is?

There's some background in one of our phenotype ontology tickets - https://github.com/pombase/fypo/issues/3005

**Comment by hdrabkin** [comment]
Is it by some chemical gradient?

**Comment by hdrabkin** [comment]
See for example http://rappel.ucsd.edu/Publications/limits_bo.pdf

**Comment by ValWood** [comment]
not  in my case ...well there are chemical gradients involved (subsequently).

I think, (but I'm not 100% sure) that the chemical gradient keeps things in the vicinity, but the "marker"  isn't the gradient, or the process by which the gradient are sensed. That's independent.

**Comment by hdrabkin** [comment]
But it could be as a response to some concentration of something. In one of the papers, setting up the premise that a protein phosphoryated near the cell membrane then diffuses away and then may encounter a phosphatase that removes the phosphate. So then if some other process depends on the level of one of the two forms (lets say the +P form), then it's activity will be greater where there is more +P form than P form, essentially setting up two regions of the cell based on that gradient.

BUt then  I guess I'm also thinking about old memories of unequal cleavage during embryogenesis; the gradient existing in the egg prior to the first cleavages. So what determined the non-uniform cytoplasm in the oocyte to begin with

**Comment by ValWood** [comment]
we do have examples of this
https://www.ncbi.nlm.nih.gov/pubmed/19474789
that we could annotate as "gradient sensors or cell size sensor" but I just captured the subsequent signalling for this.

Actually now I think about it, there must be some sort of gradient set up if the microtubules grow towards the tip. But I don't thing the "cell end marker" itself is forming the gradient. 
For now, I'm happy to ignore my request about the "cell end marker" until I have more  data. I just wondered if was something we had already addressed but I didn't know where to look.

**Comment by hdrabkin** [comment]
No, I'd be more interested in how the gradient formed. I'm sure this must have been studied in Xenopus oocytes...

**Comment by pgaudet** [comment]
@ValWood 
Any action required here?
